packages/office-ui-fabric-react/src/components/Icon/Icon.base.tsx

```diff
@@ -36,7 +36,7 @@ export class IconBase extends React.Component<IIconProps, IIconState> {
       isPlaceholder
     });
 
-    const RootType = isImage ? 'div' : 'i';
+    const RootType = isImage ? 'span' : 'i';
```

Comment:
Wont this be considered a breaking change?

Reply:
if the root style is explicit about `display` css setting being `inline-block` or `block`, I *think* it would be mostly ok to have this. I think it is a larger majority of cases that icons are hosted within `button` elements, and given this html standards violation would come up any time image icons are used, it seems like a reasonable idea to do this.

Reply:
Makes sense!